crates/cext/src/transpiler/passes/vf2.rs

```diff
@@ -118,40 +120,180 @@ pub unsafe extern "C" fn qk_vf2_layout_result_free(layout: *mut VF2LayoutResult)
     }
 }
 
+/// A set of configurations for the VF2 layout passes.
+///
+/// See the setter methods associated with this `struct` for the available configurations.
+pub struct VF2LayoutConfiguration(Vf2PassConfiguration);
+/// @ingroup QkVF2LayoutConfiguration
+/// Create a new configuration for the VF2 passes that runs everything completely unbounded.
+///
+/// Call ``qk_vf2_layout_configuration_free`` with the return value to free the memory when done.
+#[no_mangle]
+#[cfg(feature = "cbinding")]
+pub extern "C" fn qk_vf2_layout_configuration_new() -> *mut VF2LayoutConfiguration {
+    Box::into_raw(Box::new(VF2LayoutConfiguration(
+        Vf2PassConfiguration::default_unbounded(),
+    )))
+}
+/// @ingroup QkVF2LayoutConfiguration
+/// Create a new configuration for the VF2 passes that runs everything completely unbounded.
+///
+/// # Safety
+///
+/// Behavior is undefined if ``config`` is a non-null pointer, but does not point to a valid,
+/// aligned ``VF2LayoutConfiguration`` object.
+#[no_mangle]
+#[cfg(feature = "cbinding")]
+pub unsafe extern "C" fn qk_vf2_layout_configuration_free(config: *mut VF2LayoutConfiguration) {
+    if !config.is_null() {
+        if !config.is_aligned() {
+            panic!("Attempted to free a non-aligned pointer.");
+        }
+        let _ = unsafe { Box::from_raw(config) };
```

Comment:
Some `unsafe` lines here are missing a `// SAFETY:` comment (though I also realize that this whole file misses a lot of these 😄)

Reply:
Thanks - I must have forgot while I was dealing with the big merge conflict around this file.